doc/requirements.rst

```diff
@@ -0,0 +1,119 @@
+=====================
+Software Requirements
+=====================
+
+.. contents:: `Contents`
+    :depth: 3
+    :local:
+
+---------------------------------
+Requirements for mlx.traceability
+---------------------------------
+
+.. item:: RQT-TRACEABILITY A plugin for sphinx documentation system, adding traceability
+
+    System shall implement a plugin for the sphinx documentation system. It shall add tracebility within
+    the documentation.
+
+.. item:: RQT-ITEMIZE Allow splitting the documentation in parts
+    :depends_on: RQT-TRACEABILITY
+
+    The plugin shall allow for splitting the documentation in parts.
+
+.. item:: RQT-DOCUMENTATION_ID Identification of documentation part
+    :depends_on: RQT-ITEMIZE
+
+    A documentation part shall have a unique identification.
+
+.. item:: RQT-CAPTION Brief description of documentation part
+    :depends_on: RQT-ITEMIZE
+
+    A documentation part shall have a optional brief description.
+
+.. item:: RQT-CONTENT Conten of documentation part
+    :depends_on: RQT-ITEMIZE
+
+    A documentation part shall have optional content. The content shall be parseable RST, and passed
```

Comment:
write out RST here to get more hits. Maybe `reStructured text (rst)` is a good keyword bait

Reply:
rewording in 792c66b

Reply:
Nice